packages/util-retry/src/StandardRetryStrategy.ts

```diff
@@ -20,6 +20,7 @@ export class StandardRetryStrategy implements RetryStrategyV2 {
   }
 
   public async acquireInitialRetryToken(retryTokenScope: string): Promise<StandardRetryToken> {
+    this.retryToken.resetRetryCount();
     return this.retryToken;
```

Comment:
I considered returning a new default token here, but that would reset the retry token count of 500.

Reply:
this now returns a new default token but with a pointer to the shared token availability counter.